<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Admin-PC" ignored="0" name="test1" tests="1" failures="1" timestamp="20 Feb 2019 03:58:27 GMT" time="55.595" errors="0">
  <testcase name="loginTest" time="31.378" classname="com.crm.qa.testcases.test.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [CRMPRO1] but found [CRMPRO]">
      <![CDATA[java.lang.AssertionError: expected [CRMPRO1] but found [CRMPRO]
at com.crm.qa.testcases.test.LoginTest.loginTest(LoginTest.java:37)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- test1 -->
